beacon_node/eth2-libp2p/src/behaviour.rs

```diff
@@ -79,7 +79,7 @@ impl<TSubstream: AsyncRead + AsyncWrite> Behaviour<TSubstream> {
             discovery: Discovery::new(local_key, net_conf, log)?,
             ping: Ping::new(ping_config),
             identify,
-            seen_gossip_messages: LruCache::new(256),
+            seen_gossip_messages: LruCache::new(100_000),
```

Comment:
What's the impact on memory usage? The LRU cache probably stores a `u64` for each key, and the value is `()` (1 byte?) so maybe 900KB? Manageable.

Reply:
Maybe with alignment the `()` values take a whole word, in which case maybe 1.6MB, which is still fine

Reply:
Actually, it looks like `lru::LruCache` stores the full keys... https://docs.rs/lru/0.4.3/src/lru/lib.rs.html#140-171